subsys/bluetooth/audio/bap_base.c

```diff
@@ -0,0 +1,562 @@
+/* bap_base.c - BAP BASE handling */
+
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/bluetooth/iso.h>
+#include <zephyr/bluetooth/uuid.h>
+#include <zephyr/bluetooth/audio/bap.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/check.h>
+
+LOG_MODULE_REGISTER(bt_bap_base, CONFIG_BT_BAP_BASE_LOG_LEVEL);
+
+#define BASE_MAX_SIZE          (UINT8_MAX - 1 /* type */ - BT_UUID_SIZE_16)
+#define BASE_SUBGROUP_MAX_SIZE (BASE_MAX_SIZE - 3 /* pd*/ - 1 /* subgroup count */)
+#define BASE_SUBGROUP_MIN_SIZE                                                                     \
+	(1 /* num bis*/ + 5 /* codec ID*/ + 1 /* cc_len */ + 1 /*meta len*/ + 1 /* BIS index*/ +   \
```

Comment:
does the formatting guidelines say something about spaces in comments?

Reply:
No, but I can certainly see that I'm all sorts of inconsistent here :D